docs/fastdds/xml_configuration/domainparticipant.rst

```diff
@@ -0,0 +1,397 @@
+|                                 | and period of the initial DomainParticipant's  |                     |             |
+|                                 | |br| discovery messages.                       |                     |             |
+|                                 | See :ref:`Initial Announcements`.              |                     |             |
++---------------------------------+------------------------------------------------+---------------------+-------------+
+| ``<staticEndpointXMLFilename>`` | The XML filename with the static EDP |br|      | ``string``          |             |
+|                                 | configuration. Only necessary if |br|          |                     |             |
+|                                 | the ``<EDP>`` member is set to |br|            |                     |             |
+|                                 | ``STATIC``. See :ref:`discovery_static`.       |                     |             |
++---------------------------------+------------------------------------------------+---------------------+-------------+
+
+.. _partfiltering:
+
+ignoreParticipantFlags
+######################
+
++--------------------------+-------------------------------------------------------------------------------------------+
+| Possible values          | Description                                                                               |
++==========================+===========================================================================================+
+| NO_FILTER                | All Discovery traffic is processed.                                                       |
++--------------------------+-------------------------------------------------------------------------------------------+
+| FILTER_DIFFERENT_HOST    | Discovery traffic from another host is discarded.                                         |
++--------------------------+-------------------------------------------------------------------------------------------+
+| FILTER_DIFFERENT_PROCESS | Discovery traffic from another process on the same host is discarded.                     |
++--------------------------+-------------------------------------------------------------------------------------------+
+| FILTER_SAME_PROCESS      | Discovery traffic from DomainParticipant's own process is discarded.                      |
++--------------------------+-------------------------------------------------------------------------------------------+
+| FILTER_DIFFERENT_PROCESS | Discovery traffic from DomainParticipant's own host is discarded.                         |
++--------------------------+                                                                                           |
+| FILTER_SAME_PROCESS      |                                                                                           |
```

Comment:
This should be `FILTER_DIFFERENT_PROCESS | FILTER_SAME_PROCESS`. Update also in discovery